src/main/webapp/index.html

```diff
@@ -0,0 +1,28 @@
+
+<html>
+    <body>
+        <h1>Welcome To Your Open Liberty Microservice</h1>
+        <div>
+            <h1>Further Reading</h1>
+            <p>Please see below for information on related technologies.</p>
+        </div>
+        <div>
+            <h2>Open Liberty</h2>
+            <p>
+                The <a href="http://openliberty.io/">Open Liberty project</a> is an open
+                community supporting Java EE and <a href="http://microprofile.io/">MicroProfile</a>, a runtime starting fast and made for running cloud-native microservices.
+            </p>
+        </div>
+        <div>
+            <h2>Odo</h2>
+            <p>
+                The <a href="http://odo.dev/">odo CLI tool </a> is, iterative, and straightforward way for developers who write, build, and deploy applications on Kubernetes and OpenShift.
```

Comment:
is, iterative, and straightforward way  =>
is an iterative and straightforward way

who write, build, and deploy =?
to write, build, and deploy

Reply:
Thanks Scott. I updated it to reflect this and the reset of your comments.